Casks/f/filen.rb

```diff
@@ -2,7 +2,8 @@
   arch arm: "arm64", intel: "x64"
 
   version "3.0.40"
-  sha256 :no_check
+  sha256 arm:   "dfb0a0c11e07a97567785c6be5fc53142256cdd68eb4fd6f9bb30a531ec94fc5",
+         intel: "4a5b644c632e49d0abe65957bb7a513ebb83753acfd1b8978af29602e085c6c9"
```

Comment:
```suggestion
  sha256 :no_check
```

We need to use `:no_check` because the cask's `url` is unversioned. Otherwise, this change would cause checksum mismatches between new releases and version bumps.

Reply:
Got it.